### ISSUE TITLE
No issue, just love ❤️ 

### ISSUE THREAD

**Issue by Brototype** [alberti42/Volume-Control#3]
Thank you so much, this solved all my volume control issues in my streaming-to-sonos-in-the-office-room-situation. Keep it up! ❤️

**Comment by jorisw** [comment]
Agreed, from a Spotify Connect user! Great app, and the first to solve this problem.

**Comment by jonbhanson** [comment]
Thanks for the great solution! You rock!

**Comment by alberti42** [comment]
Thank you! Very much appreciated. I have uploaded a new version 1.7.2 which
better displays the icon in the menu bar and fixes a few other issues.

Best, Andrea

**Comment by jlippold** [comment]
Cosigning, thanks for this app

**Comment by dvtoever** [comment]
Excellent tool! Have been looking for a solution like this before but gave up. This works perfect!

**Comment by gnakh** [comment]
Definitively a must-have. Danke schön!
